examples/llava/clip.cpp

```diff
@@ -488,12 +500,34 @@ struct clip_vision_model {
     struct ggml_tensor * mm_model_mlp_2_b;
     struct ggml_tensor * mm_model_peg_0_w;
     struct ggml_tensor * mm_model_peg_0_b;
+
+    // MINICPMV projection
+    // struct ggml_tensor * mm_model_pos_embed;
```

Comment:
(nitpick) This should be removed

Reply:
ok. 
But I have to explain that this place is not invalid code. This variable is part of minicpmv2.0 that failed to be merged.
^_^